include/h2o.h

```diff
@@ -1968,6 +1851,7 @@ typedef struct st_h2o_proxy_config_vars_t {
     uint64_t io_timeout;
     uint64_t connect_timeout;
     uint64_t first_byte_timeout;
+    uint64_t keepalive_timeout;
```

Comment:
Is the intent here to have a timeout specific to HTTP/2? I ask this because IIUC timeout of HTTP/1 client would be handled using the timeout of HTTP/1.

If that is the case, would it make sense to rename it to h2_keepalive_timeout?

Reply:
No, this is also used for HTTP/1 at https://github.com/h2o/h2o/blob/1d602e58f2aaf864cd3900a9ccf634910856f522/lib/handler/configurator/proxy.c#L371

Reply:
Ah! Thank you for the clarification.